src/masoniteorm/config.py

```diff
@@ -21,3 +22,95 @@ def load_config(config_path=None):
             f"ORM configuration file has not been found in {selected_config_path}."
         )
     return config_module
+
+
+def db_url(database_url=None, prefix="", options={}, log_queries=False):
+    """Parse connection configuration from database url format. If no url is provided,
+    DATABASE_URL environment variable will be used instead.
+
+    Reference: Code adapted from https://github.com/jacobian/dj-database-url
+    """
+
+    url = database_url or os.getenv("DATABASE_URL")
+    if not url:
+        raise InvalidUrlConfiguration("Database url is empty !")
+
+    # Register database schemes in URLs.
+    urlparse.uses_netloc.append("postgres")
+    urlparse.uses_netloc.append("postgresql")
+    urlparse.uses_netloc.append("pgsql")
+    urlparse.uses_netloc.append("postgis")
+    urlparse.uses_netloc.append("mysql")
+    urlparse.uses_netloc.append("mysql2")
+    urlparse.uses_netloc.append("mysqlgis")
+    urlparse.uses_netloc.append("mssql")
+    urlparse.uses_netloc.append("sqlite")
+
+    DRIVERS_MAP = {
+        "postgres": "postgres",
+        "postgresql": "postgres",
+        "pgsql": "postgres",
+        "postgis": "postgres",
+        "mysql": "mysql",
+        "mysql2": "mysql",
+        "mysqlgis": "mysql",
+        "mysql-connector": "mysql",
+        "mssql": "mssql",
+        "sqlite": "sqlite",
+    }
+
+    # this is a special case, because if we pass this URL into
+    # urlparse, urlparse will choke trying to interpret "memory"
+    # as a port number
+    if url == "sqlite://:memory:":
```

Comment:
We could probably check for a couple of string here. Maybe also just `:memory:` too

Reply:
Nobody is going to remember this string without referring to the docs

Reply:
Yeah we could check just `:memory:`.  What do you mean by couple of strings lol ?

Reply:
```python
if url in ("sqlite://:memory:", ":memory:")
```

I am also assuming that `sqlite://:memory:` is the officially supported string. I just want to be able to remember a string and I'm not going to remember `sqlite://:memory:` lol

Reply:
I'll forget one of the 3 `:` and pull my hair out why its not working

Reply:
Oh okay got it. Actually you can already go the simple way`sqlite://` but I can add `sqlite://memory` also.

Reply:
Now you can use those for SQLite in memory:
```python
db_url("sqlite://")
db_url("sqlite://memory")
db_url("sqlite://:memory:")  # the official one
```
And for a SQLite classic db file:
```python
db_url("sqlite://orm.sqlite3")
```